src/stripe/stripe-service.ts

```diff
@@ -176,16 +209,15 @@ export class StripeService {
     });
   }
 
-  static isStripeHandledCardError(error: StripeError) {
-    if (
+  static isStripeHandledFormError(error: StripeError) {
+    const isValidationError = error.type === "validation_error";
```

Comment:
This introduces a small behavioral change: an error message will no longer be shown after closing the wallet’s native prompt.